crates/autopilot/src/domain/settlement/solution/mod.rs

```diff
@@ -313,4 +313,58 @@ mod tests {
             eth::U256::from(6890975030480504u128)
         );
     }
+
+    // https://etherscan.io/tx/0x688508eb59bd20dc8c0d7c0c0b01200865822c889f0fcef10113e28202783243
+    #[test]
+    fn testing_() {
```

Comment:
will remove test before merging

Reply:
I think you can even leave this checked in as a regression test if it's properly named.

Reply:
fixed